src/v2/view-builder/views/device/DeviceChallengePollView.js

```diff
@@ -9,6 +9,8 @@ import { doChallenge } from '../../utils/ChallengeViewUtil';
 import OktaVerifyAuthenticatorHeader from '../../components/OktaVerifyAuthenticatorHeader';
 import { getSignOutLink } from '../../utils/LinksUtil';
 
+const INVALIDATED_ACCOUNT_DEVICE_ERR_MSG = 'Your device or account was invalidated';
```

Comment:
In the idx error response there must be a `i18nKey` that should exists.. that can be used to identify the error instead of using error message itself.

Reply:
Updated